lib/mrsk/configuration.rb

```diff
@@ -252,10 +252,8 @@ def role_names
 
     def git_version
       @git_version ||=
-        if system("git rev-parse")
-          uncommitted_suffix = Mrsk::Utils.uncommitted_changes.present? ? "_uncommitted_#{SecureRandom.hex(8)}" : ""
-
-          "#{`git rev-parse HEAD`.strip}#{uncommitted_suffix}"
+        if system("git describe")
+          `git describe --always --dirty='-dirty-#{SecureRandom.hex(8)}`
```

Comment:
`git describe` doesn't treat untracked files as dirty, could we check for them and append the `dirty-` suffix manually?